client/sites-dashboard/components/sites-grid-item.tsx

```diff
@@ -109,14 +110,23 @@ export const SitesGridItem = memo( ( props: SitesGridItemProps ) => {
 	const isWpcomStagingSite = isStagingSite( site );
 	const translatedStatus = useSiteLaunchStatusLabel( site );
 	const isTrialSitePlan = useSelector( ( state ) => isTrialSite( state, site.ID ) );
+	const isWpAdminDefaultSite = useSelector( ( state ) =>
```

Comment:
We generally use `is*` for boolean values, but this seems to be a string value. I think it should be updated accrodingly.

Reply:
Solved!